File: public/public-web/src/test/java/ch/difty/scipamato/publ/web/resources/MetaOTCssResourceReferenceTest.java
package ch.difty.scipamato.publ.web.resources;

import static org.assertj.core.api.Assertions.assertThat;

import org.junit.jupiter.api.Test;

class MetaOTCssResourceReferenceTest {

    private static final MetaOTCssResourceReference REF = MetaOTCssResourceReference.get();

    @Test
    void canGetInstance() {
        assertThat(REF).isInstanceOf(MetaOTCssResourceReference.class);
    }

    @Test
    void assertResourceName() {
        assertThat(REF.getName()).isEqualTo("css/MetaOT.css");
    }
}

File: public/public-web/src/test/kotlin/ch/difty/scipamato/publ/web/resources/MetaOTCssResourceReferenceTest.kt
package ch.difty.scipamato.publ.web.resources

import org.amshove.kluent.shouldBeEqualTo
import org.amshove.kluent.shouldBeInstanceOf
import org.junit.jupiter.api.Test

internal class MetaOTCssResourceReferenceTest {

    @Test
    fun canGetInstance() {
        MetaOTCssResourceReference shouldBeInstanceOf MetaOTCssResourceReference::class
    }

    @Test
    fun assertResourceName() {
        MetaOTCssResourceReference.name shouldBeEqualTo "css/MetaOT.css"
    }
}
